core/server/themes/active.js

```diff
@@ -84,6 +92,30 @@ class ActiveTheme {
     config(key) {
         return this._config[key];
     }
+
+    mount(blogApp) {
+        let hbsOptions = {
+            partialsDir: [config.get('paths').helperTemplates],
+            onCompile: function onCompile(exhbs, source) {
+                return exhbs.handlebars.compile(source, {preventIndent: true});
+            }
+        };
+
+        if (this.hasPartials()) {
+            hbsOptions.partialsDir.push(this.partialsPath);
+        }
+
+        // reset the asset hash
+        // @TODO: set this on the theme instead of globally, or use proper file-based hash
+        config.set('assetHash', null);
```

Comment:
> set this on the theme instead

The active theme constructor generates a new theme asset hash.
This should auto resolve this issue, because on each theme activation a new active theme class get's created.